resources/views/community/components/user/profile-meta.blade.php

```diff
@@ -0,0 +1,68 @@
+@props([
+    'averageCompletionPercentage' => '0.0',
+    'averageFinishedGames' => 0,
+    'averagePointsPerWeek' => 0,
+    'developerStats' => [],
+    'hardcoreRankMeta' => [],
+    'recentPointsEarned' => [],
+    'socialStats' => [],
+    'softcoreRankMeta' => [],
+    'totalHardcoreAchievements' => 0,
+    'totalSoftcoreAchievements' => 0,
+    'userClaims' => null, // ?array
+    'userMassData' => [],
+    'username' => '',
+])
+
+<?php
+use App\Site\Enums\Permissions;
+
+$jrDevPermission = Permissions::JuniorDeveloper;
+?>
+
+<div class="relative">
+    <x-user.profile.primary-meta
+        :hardcoreRankMeta="$hardcoreRankMeta"
+        :softcoreRankMeta="$softcoreRankMeta"
+        :userMassData="$userMassData"
+        :username="$username"
+    />
+
+    <div class="mb-2 sm:ml-[142px] md:ml-0 md:absolute md:top-0.5 md:right-0 lg:relative lg:ml-[142px] xl:absolute xl:ml-0">
+        <x-user.profile.social-interactivity :username="$username" />
+    </div>
+</div>
+
+<hr class="border-embed-highlight mb-2" />
+
+<x-user.profile.follows-you-label :username="$username" />
```

Comment:
This seems out of place where it's at. Could it be moved up near the follow widget?

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/a892caca-6dc5-493d-a29c-d0f702da78d2)

Reply:
I've spent some time tweaking this label and the 3 social buttons across all breakpoints. Hopefully it's a little less out of place now.